README.md

```diff
@@ -1,68 +1,73 @@
 # Google Cloud Spanner PostgreSQL Adapter
 
-## Introduction
-Spanner-PGAdapter is a simple, MITM, forward, non-transparent proxy, which 
-translates Postgres wire protocol into the Cloud Spanner equivalent. By running
-this proxy locally, any Postgres client (including the SQL command-line client
-PSQL) should function seamlessly by simply pointing its outbound port to the 
-this proxy's inbound port. For the time being, we only support PSQL versions 11, 12, 13, and 14.
-
-Additionally to translation, this proxy also concerns itself with authentication
-and to some extent, connection pooling. Translation for the most part is simply
-a transformation of the [PostgreSQL wire protocol](https://www.postgresql.org/docs/8.2/protocol-message-formats.html)
-except for some cases concerning PSQL, wherein the query itself is translated.
+Google Cloud Spanner PGAdapter is a proxy which translates the PostgreSQL wire-protocol into the
+Cloud Spanner equivalent for Cloud Spanner databases [that use the PostgreSQL dialect](https://cloud.google.com/spanner/docs/postgresql-interface).
 
-Simple query mode and extended query mode are supported, and any data type
-supported by Spanner is also supported. Items, tables and language not native to
-Spanner are not supported, unless otherwise specified.
+PGAdapter has been tested and can be used with the following clients:
+1. `psql`: Versions 11, 12, 13 and 14 are supported. See [psql support](docs/psql.md) for more details.
+2. `JDBC`: Versions 42.x and higher have __limited support__. See [JDBC support](docs/jdbc.md) for more details.
+3. `pgx`: Version 4.15 and higher have __limited support__. See [pgx support](docs/pgx.md) for more details.
 
-Though the majority of functionality inherent in most PostgreSQL clients
-(including PSQL and JDBC) are included out of the box, the following items are
-not supported:
-* Functions
-* COPY <table_name> TO ...
-* COPY <table_name> FROM <filename | PROGRAM program>
-* Prepared Statement DESCRIBE
-* SSL
-* PSQL meta-commands not included in this list (i.e.: these are supported):
-  * `\d <table>` 
-  * `\dt <table>`
-  * `\dn <table>`
-  * `\di <table>`
-  * `\l`
-
-COPY <table_name> FROM STDIN is supported.
+Other drivers and tools have __not been specifically tested__ and are not known to work with PGAdapter
+and are therefore __not supported__. Drivers and tools that use [simple query protocol](https://www.postgresql.org/docs/current/protocol-flow.html#id-1.10.5.7.4)
+are likely to work with PGAdapter. Drivers and tools that use the [extended query protocol](https://www.postgresql.org/docs/current/protocol-flow.html#PROTOCOL-FLOW-EXT-QUERY)
+are more likely to at least have some issues when used with PGAdapter.
```

Comment:
I've removed the section. I agree that it might be a little confusing.

The situation is that the simple query protocol is, as the name already says, simple. This means that we already support a lot more of that protocol than the extended protocol, which means that tools that use that protocol by default are a lot more likely to work out of the box than tools that use the extended mode. But it's probably better not to advertise that here, as it could confuse users, especially those who do not know what the simple/extended protocol is. Those who do know the difference, might automatically be inclined to try it out.